### ISSUE TITLE
Charset: support for pre-HTML5 meta syntax

### ISSUE THREAD

**Issue by chicgeek** [ffoodd/a11y.css#311]
While working on a CakePHP 2 site I noticed that it output the [lengthier format](https://book.cakephp.org/2.0/en/core-libraries/helpers/html.html#HtmlHelper::charset) for meta charset definition which raised an error with a11y.css. According to MDN, this is still valid syntax though it is not preferred:

> The <meta> element with a charset attribute is a synonym for the pre-HTML5 `<meta http-equiv="Content-Type" content="text/html; charset=IANAcharset">`, where IANAcharset contains the value of the equivalent charset attribute. This syntax is still allowed, although no longer recommended. [Source](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta)

I'm not sure what the plans are for a11y.css's compatibility with older (yet still-valid) syntax. Is this something that should be included (I can raise a PR) or is pre-HTML5 out of scope?

**Comment by ffoodd** [comment]
This is a nice one, thanks. :)

Despite the fact that a11y.css is meant to be applied on HTML5, the test triggered is probably the "Charset should come first" one. Since this is a declared charset, the error we intend to trigger doesn't exist since you're effectiveley specifiying a charset.

To me, this syntax should raise an obsolete message, not an error.

So I see two different tasks here:

- [x] Allow the `htt-equiv` syntax on the "Charset should come first" test;
- [ ] Add some more obsolete entries dedicated to `http-equiv`, since reading the MDN docs there are other values out there we're not targeting;
- [ ] Mention in the wiki that a11y.css is HTML5 oriented (should already exists since we have an entire file for obsolete stuff, we should have specified the language version aimed as a starting point).

Thanks a lot for the issue and proposing a PR but I'm already on it :)

**Comment by ffoodd** [comment]
Here it is.
The next minor version will include the fix, but you may check it already by using the bookmarklet.

Tell me if that does the trick, and thanks again :)

**Comment by chicgeek** [comment]
Amazing. And so quick! Three gold stars and fifteen internet points to you.

(I managed to turn off my GH notifications and forgot about this until now – sorry for the late reply.)
